### PR TITLE
[#110] Print executing command

### PR DESCRIPTION
**Pull request by willbasky**
Resolves [#110](https://github.com/kowainik/summoner/issues/110)

### CODE REVIEW

**Comment by ChShersh** [comment]
@willbasky Also, comment message should be `Resolves` instead of `Resolving`. Unfortunately, GitHub doesn't recognise all words.

**Comment by ChShersh** [comment]
Okay, I tested it locally. It works as expected!

File: src/Summoner/Process.hs [hunk 1]
```diff
@@ -22,8 +22,10 @@ import Summoner.Ansi (errorMessage)
 
 -- This is needed to be able to call commands by writing strings.
 instance (a ~ Text, b ~ ()) => IsString ([a] -> IO b) where
-    fromString "cd" [arg] = setCurrentDirectory $ toString arg
-    fromString cmd args   = callProcess cmd (map toString args)
+    fromString cmd args = do
+        let cmdStr = showCommandForUser cmd (map toString args)
+        putStrLn $ "⚙  " ++ cmdStr
+        callCommand cmdStr
```
Comment (ChShersh) [inline src/Summoner/Process.hs]: You should also update changelog 🙂

Comment (willbasky) [reply src/Summoner/Process.hs]: ops! Sure..

**Pull request merged**